assets/js/components/user-input/UserInputQuestionnaire.js

```diff
@@ -343,47 +313,10 @@ export default function UserInputQuestionnaire() {
 					error={ error }
 				>
 					<UserInputSelectOptions
-						isActive={
-							activeSlug === USER_INPUT_QUESTION_HELP_NEEDED
-						}
-						slug={ USER_INPUT_QUESTION_HELP_NEEDED }
-						max={ 3 }
-						options={ USER_INPUT_ANSWERS_HELP_NEEDED }
-						next={ nextCallback }
-					/>
-				</UserInputQuestionWrapper>
-			) }
-
-			{ activeSlugIndex <=
-				steps.indexOf( USER_INPUT_QUESTION_SEARCH_TERMS ) && (
-				<UserInputQuestionWrapper
-					slug={ USER_INPUT_QUESTION_SEARCH_TERMS }
-					isActive={ activeSlug === USER_INPUT_QUESTION_SEARCH_TERMS }
-					questionNumber={ 5 }
-					title={ __(
-						'To help us identify opportunities for your site, enter the top three search terms that best describe your site’s content',
-						'google-site-kit'
-					) }
-					description={ __(
-						'Site Kit will keep you informed if people start finding you in Search for these terms',
-						'google-site-kit'
-					) }
-					next={ nextCallback }
-					nextLabel={
-						nextLabel === undefined
-							? __( 'Preview', 'google-site-kit' )
-							: nextLabel
-					}
-					back={ backCallback }
-					error={ error }
-					allowEmptyValues
-				>
-					<UserInputKeywords
-						isActive={
-							activeSlug === USER_INPUT_QUESTION_SEARCH_TERMS
-						}
-						slug={ USER_INPUT_QUESTION_SEARCH_TERMS }
-						max={ 3 }
+						isActive={ activeSlug === USER_INPUT_QUESTIONS_GOALS }
+						slug={ USER_INPUT_QUESTIONS_GOALS }
+						max={ 4 }
```

Comment:
Looks like this used to have a max of 2, not sure why it changed?

Reply:
It was me trying to check if 'other' option makes difference.
According to designs, it is up to 3 answers -> https://www.figma.com/file/CdtrHB0HAjfGRWTnTMMQI0/SummaryWidget?node-id=2101%3A61555
I am changing it now.